tensorflow/python/BUILD

```diff
@@ -496,7 +496,11 @@ tf_python_pybind_extension(
         "//tensorflow/python/lib/core:pybind11_status",
         # The headers here cannot be brought in via cc_header_only_library
         "//tensorflow/compiler/aot:llvm_targets",
-    ],
+    ] + select({
+        "@platforms//cpu:s390x": ["@llvm-project//llvm:TargetParser"],
+        "//conditions:default": [],
+    }),
+)
```

Comment:
```suggestion
```

Reply:
Fixed and rebase on top of current master